tests/search/includes/classes/test-class-search.php

```diff
@@ -3270,7 +3280,7 @@ protected static function get_property( $name ) {
 	}
 
 	public function mock_vip_safe_wp_remote_request() {
-
```

Comment:
This empty method is gross. Doesn't actually need to exist?

Reply:
No idea, that was not my code… I try not to do any refactoring during PHPCS fixes, because this complicates the PR. I will do another PR afterward if this can be fixed.

Reply:
that's a good policy